addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -12,16 +12,17 @@
                     placeholder="Property Name"
                     t-model="state.propertyDefinition.string"
                     t-on-change="onPropertyLabelChange"
+                    t-on-keypress="onPropertyLabelKeypress"
                 />
                 <div class="d-flex flex-row me-3">
                     <button
-                        class="btn btn-link p-0 m-0 me-2"
-                        t-on-click="() => props.onPropertyMove('up')">
+                        t-attf-class="btn btn-link p-0 m-0 me-2 {{state.propertyIndex === 0 ? 'disabled' : ''}}"
```

Comment:
Can't we have a ``first`` / ``last`` ? Seems  a bit low level to have that kind of comparison in xml :)